python/engine/niah_verify.py

```
@@ -0,0 +1,486 @@
+            chunk_target = [vocab] * seq_len
+            # Fill real targets where available
+            if pos + 1 < len(sequence):
+                real_targets = sequence[pos + 1:chunk_end]
+                for j, t in enumerate(real_targets):
+                    chunk_target[j] = t
+        else:
+            chunk_input = sequence[pos:chunk_end]
+            if chunk_end < len(sequence):
+                chunk_target = sequence[pos + 1:chunk_end + 1]
+            else:
+                chunk_target = sequence[pos + 1:chunk_end] + [vocab]
```

Ruff (0.15.5)

[warning] Consider `[*sequence[pos + 1:chunk_end], vocab]` instead of concatenation

Replace with `[*sequence[pos + 1:chunk_end], vocab]`

(RUF005)